packages/tokens/src/fonts.ts

```diff
@@ -0,0 +1,6 @@
+const fonts = {
+  default: `Akzidenz, 'Helvetica Neue', Helvetica, Arial, sans-serif`,
+  code: `'Source Code Pro', monospace`,
```

Comment:
We should have a more robust set of backups for monospaced fonts – `'Source Code Pro', Menlo, monospace` would probably be better here.

Reply:
👍